app.authpass.AuthPass.yaml

```diff
@@ -0,0 +1,61 @@
+app-id: app.authpass.AuthPass
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+
+command: authpass
+finish-args:
+  # Filesystem and network access
+  - --filesystem=home
+  - --share=network
+  # Display access
+  - --socket=wayland
+  - --share=ipc
+  - --socket=fallback-x11
+  # Keyring access
+  - --talk-name=org.freedesktop.secrets
+  # OpenGL rendering
+  - --device=dri
+
+cleanup:
+  - /include
```

Comment:
no `*.a` left behind?

Reply:
Not sure if I fully understand that `cleanup` property.. but i guess the only thing which could leave anything behind is the `keybinder` build.. i've searched through the repo and found e.g. https://github.com/flathub/io.github.Pithos/blob/master/io.github.Pithos.json which uses

```
"cleanup": ["/lib/*.la", "/include", "/share", "/lib/pkgconfig"]
```

(although [others don't use any cleanup at all](https://github.com/flathub/com.github.KRTirtho.Spotube/blob/master/com.github.KRTirtho.Spotube.yml) 🤷‍♂️)